src/theory/theory_engine.cpp

```diff
@@ -749,34 +749,6 @@ bool TheoryEngine::presolve() {
   return false;
 }/* TheoryEngine::presolve() */
 
-void TheoryEngine::postsolve() {
-  // no longer in SAT mode
-  d_inSatMode = false;
```

Comment:
So the one change in this PR is that these flags are not reset, right?

Are these reset somewhere else?